src/components/Review/DecisionArea.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import { Button, Header, Modal, Radio, Segment, TextArea, TextAreaProps } from 'semantic-ui-react'
+import { SummaryViewWrapper } from '../../formElementPlugins'
+import { ReviewResponseDecision } from '../../utils/generated/graphql'
+import { DecisionAreaState, ReviewQuestionDecision } from '../../utils/types'
+import strings from '../../utils/constants'
+
+interface DecisionAreaProps {
+  state: DecisionAreaState
+  setDecision: Function
+  submitHandler: (_: any) => void
+  problemMessage: string
+  setProblemMessage: Function
+}
+
+const DecisionArea: React.FC<DecisionAreaProps> = ({
+  state,
+  setDecision,
+  submitHandler,
+  problemMessage,
+  setProblemMessage,
+}) => {
+  const { open, review, summaryViewProps } = state
+  const handleChange = (value: ReviewResponseDecision) => {
+    const { id, comment } = review as ReviewQuestionDecision
+    setProblemMessage('')
+    setDecision({
+      ...state,
+      review: { id, comment, decision: value },
+    })
+  }
+
+  const handleUpdateComment = (_: any, { value }: TextAreaProps) => {
+    const { review } = state
+    setDecision({ ...state, review: { ...review, comment: value } })
+  }
+
+  return (
+    <Modal
+      closeIcon
+      open={open}
+      onClose={() => setDecision({ open: false })}
+      size="fullscreen"
+      style={{ margin: 0, background: 'transparent' }}
```

Comment:
I wonder if the dark background is too dark. I was thinking we should be able to read more of the underneath content. Not sure though, might look bad.

Reply:
Any style shouldn't be done yet - as advised by @andreievg 

So probably make a note on #78 to improve that please.